### PR TITLE
Add documents for modules:

### PR DESCRIPTION
**Pull request by winniex1**
metric_handler.py, package_manager.py, metric.py, registered_metric.py, si_unit.py, liotad.py,
cal_sha1sum.py and utility.py.

### CODE REVIEW

File: liota/entities/metrics/metric.py [hunk 1]
```diff
@@ -61,4 +61,12 @@ def __init__(self, name, entity_type="Metric",
         self.sampling_function = sampling_function
```
Comment (KohliDev) [inline liota/entities/metrics/metric.py]: Comments to be added for init method

File: liota/lib/utilities/utility.py [hunk 2]
```diff
@@ -274,6 +300,8 @@ def read_liota_config(section, name):
 def read_user_config(config_file_path):
     """
     Returns the user defined configuration as a dictionary from DEFAULT section.
+    :param config_file_path: path of a configuration file
+    :return: the user defined configuration as a dictionary
     """
 
     config = ConfigParser.RawConfigParser()
```
Comment (KohliDev) [inline liota/lib/utilities/utility.py]: Add comments for valid_named_pipe

**Comment by KohliDev** [comment]
Resolved utility.py

File: liota/lib/utilities/utility.py (hunk 2 above)
Comment (winniex1) [reply liota/lib/utilities/utility.py]: done

File: liota/entities/metrics/metric.py (hunk 1 above)
Comment (winniex1) [reply liota/entities/metrics/metric.py]: done

**Pull request merged**